recipes/stringzilla/all/test_package/CMakeLists.txt

```diff
@@ -7,5 +7,9 @@ add_executable(${PROJECT_NAME} test_package.cpp)
 target_link_libraries(${PROJECT_NAME} PRIVATE stringzilla::stringzilla)
 target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_11)
 if(stringzilla_VERSION VERSION_LESS 2.0)
-  target_compile_definitions(${PROJECT_NAME} PRIVATE STRINGZILLA_LESS_2_0)
+  target_compile_definitions(${PROJECT_NAME} PRIVATE STRINGZILLA_API=1)
+elseif(stringzilla_VERSION VERSION_LESS 3.0)
+  target_compile_definitions(${PROJECT_NAME} PRIVATE STRINGZILLA_API=2)
+else()
+  target_compile_definitions(${PROJECT_NAME} PRIVATE STRINGZILLA_API=3)
```

Comment:
Could you put this variable directly in the CMakeToolchain, something like `tc.preprocessor_definitions["STRINGZILLA_API"] = Version(self.dependencies["stringzilla"].ref.version).major`?

Reply:
@Ahajha 
Thank you for your comment!
I fixed as you suggested.
Could you review it again?